layouts/partials/footer.html

```diff
@@ -11,27 +10,22 @@ <h5 class="uk-margin-small-bottom">Email</h5>
       </div>
       <div>
         <h5 class="uk-margin-small-bottom">Address</h5>
-                  <div uk-lightbox>
-                    <a href="{{.Site.Params.footer.googlemaps}}" data-caption="{{.Site.Params.footer.address}}"
-                      data-type="iframe">{{.Site.Params.footer.address}}</a>
-      </div>
+          <div uk-lightbox>
+            <a href="{{.Site.Params.footer.googlemaps}}" data-caption="{{.Site.Params.footer.address}}"
+               data-type="iframe">{{.Site.Params.footer.address}}</a>
+          </div>
       </div>
-
     </div>
 
 
-
-
-
-  <div class="footer-copyright uk-text-center" style="padding-bottom: 1rem">
-    <img loading="lazy" src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" style="height: 1.2em;
+    <div class="footer-copyright uk-text-center" style="padding-bottom: 1rem">
+      <img loading="lazy" src="https://mirrors.creativecommons.org/presskit/icons/cc.svg" style="height: 1.2em;
   width: 1.2em;" alt="Creative Commons">
-    <img loading="lazy" src="https://mirrors.creativecommons.org/presskit/icons/by.svg" style="height: 1.2em;
+      <img loading="lazy" src="https://mirrors.creativecommons.org/presskit/icons/by.svg" style="height: 1.2em;
   width: 1.2em;" alt="CC-BY">
-    2020
-    {{.Site.Params.Author}}
-
-
+      2023
```

Comment:
`{{ now.Year }}` can be used to update this on every build

```suggestion
      {{ now.Year }}
```